source/a_box/main.c

```diff
@@ -288,14 +293,13 @@ void heartBeatLED()
 void monitorStatus()
 {
    uint8_t bms_err, imd_err;
-   bms_err = orionErrors();
+   bms_err = orionErrors() | readTemps(&tmu);
```

Comment:
in this config it is possible it doesn't call readTemps if there is an orionError
Its kind of like if you do (a() && b()) and the return value of a is false, it wont run b().

Reply:
yep, split it into 2 different lines so they each get evaluated always